connectors/sources/salesforce.py

```diff
@@ -1201,7 +1163,6 @@ def map_content_document(self, content_document):
 
         return {
             "_id": content_document.get("Id"),
-            "_attachment": content_document.get("_attachment"),
```

Comment:
This will be fine. Salesforce is the only connector where the flow changed to support the new download funcs.

Before:
Download file -> map to elastic doc -> return doc

Now:
map to elastic doc -> download file -> return doc

The `_attachment` (or `body` if local extraction is used`) is added to the doc during the "download file" phase.